covid19/printer.py

```diff
@@ -35,17 +35,11 @@ def get_in_info(self, message):
 
 
 def new_lines(count=1):
-    i = 0
-    while i < count:
-        print("")
-        i += 1
+    print(end="\n"*count)
```

Comment:
I changed this in recent verion @Naaru-01 I don't know why it was failing :laughing: